src/Components/Modal.js

```diff
@@ -0,0 +1,198 @@
+import styled from 'styled-components';
+// import img from "./modal.jpg"
+// import { mdiWikipedia } from '@mdi/js';
+import {Icon} from '@iconify/react';
+
+export default function Modal({open, onClose, content}) {
+  console.log(content);
+  return (
+    open && (
+      <>
+        <Styled_Close_Button type="button" onClick={onClose}>
+          <Icon
+            icon="ant-design:close-square-outlined"
+            width="45"
+            height="45"
+          />
+        </Styled_Close_Button>
+        <StyledModalOuter>
+          <StyledModalInner>
+            <Styled_Wikipedia_Icon>
+              <Icon
+                icon="fa6-brands:wikipedia-w"
+                width="40"
+                height="40"
+                src={content.source}
+              />
+            </Styled_Wikipedia_Icon>
+            {/* images */}
+            <StyledImage src={content.spectral_img}></StyledImage>
+
+            <SteyledElement>
+              <span>{content.symbol}</span>
+            </SteyledElement>
+            <SteyledElementName>
+              <span>{content.name}</span>
+            </SteyledElementName>
+            <StyledAtomicMass>
+              <span>{content.atomic_mass}</span>
+            </StyledAtomicMass>
+
+            <Styled_Span>
+              <Styled_Icon>
+                <Icon icon="logos:react" width="25" height="25" />
+                <span>Overview</span>
+              </Styled_Icon>
+            </Styled_Span>
```

Comment:
Same as above, check the styled component names:

```suggestion
            <StyledSpan>
              <StyledIcon>
                <Icon icon="logos:react" width="25" height="25" />
                <span>Overview</span>
              </StyledIcon>
            </StyledSpan>
```